src/types.typ

```diff
@@ -1,7 +1,22 @@
-#import "types/any.typ": *
-#import "types/array.typ": *
-#import "types/dictionary.typ": *
-#import "types/logical.typ": *
-#import "types/number.typ": *
-#import "types/string.typ": *
-#import "types/tuple.typ": *
\ No newline at end of file
+#import "base-type.typ": base-type
+#import "assertions.typ": one-of
+#import "types/array.typ": array
+#import "types/dictionary.typ": dictionary
+#import "types/logical.typ": either
+#import "types/string.typ": string, ip, email
+#import "types/tuple.typ": tuple
+
+#let any = base-type.with(name: "any")
+#let boolean = base-type.with(name: "bool", types: (bool,))
+#let color = base-type.with( name: "color", types: (type(rgb(0,0,0)),)) 
```

Comment:
I think the color type can be named using [`color`](https://typst.app/docs/reference/visualize/color/).

Reply:
done #[6f79ed2](https://github.com/JamesxX/valkyrie/pull/5/commits/6f79ed25f8c7c128a1ff3fa3054f920b463a194a)